### DISCUSSION TITLE
Trying to install DoWhy package on macOS 14 Sonoma - Command failed: python python -m pip install Orange3-Text exited with non zero status

### DISCUSSION THREAD

**Discussion by dsnishimura** [biolab/orange3#6830] [Q&A] [answered]
I am trying to install the DoWhy add-in for Orange 3 but I get this error each time:

> Command failed: python python -m pip install Orange3-Text exited with non zero status.

It seems like it is some kind of conflict between Mac's Python and Orange's Python. Has someone gone through something similar, or knows how to work around this issue?

**Comment by janezd** [comment, marked as answer]
I assume that you used Orange's add-on dialog, clicked `Add more ...` and entered `dowhy`?

This won't do anything - at best. DoWhy is a Python package, not an Orange add-on. Orange will install it into its environment, but won't found any widgets inside, so it will ignore it.

I wrote *at best* because a worse scenario is that installing a package like doWhy upgrades or downgrades some of the packages used within Orange, like numpy or scipy. Orange may or may not work with the up/downgraded version. I do not understand how is this related to orange3-text add-on, but anyway I'm just guessing how you are installing doWhy and what your configuration looks like.

I tried installing doWhy. It installed, Orange still works (including the text add-on), but I now reinstalled Orange to clean up its environment, just for the case it ruined anything. I recommend you do the same. :)

**Reply by dsnishimura** [reply]
I misspelled the text add part, it was actually related to DoWhy. I apologize for that.

I will keep using DoWhy in python and try not to mess with Orange's Python enviroment.

Thanks a lot for the response!!
